samples/src/test/java/org/autorefactor/refactoring/rules/samples_out/SwitchSample.java

```diff
@@ -361,4 +361,169 @@ public void doNotReplaceWithSwitchOnPrimitiveWrapper(Integer i1) {
             i = 30;
         }
     }
+
+    public void mergeCases(int i1) {
+        int i = 0;
+        switch (i1) {
+        case 0:
+        case 1:
+            i = 0;
+            break;
+        default:
+            i = 1;
+            break;
+        }
+    }
+
+    public void mergeCasesWithUnbrokenCase(int i1) {
+        int i = 0;
+        switch (i1) {
+        case 1:
+            i = 1;
+        case 0:
```

Comment:
We may even eliminate cases which are the same as the default case?
WDYT?

I am happy if you prefer to leave this for later.